src/state-summary/state-card-climate.html

```diff
@@ -37,7 +37,7 @@
       <div class='state'>
         <div class='target'>
           <span class="operation-mode">
```

Comment:
Rename `operation-mode` class to something state-related

Reply:
Would `state-label` work?